### PR TITLE
release-1.17, fix latest version to deprecated version

### PR DESCRIPTION
**Pull request by kbhawkey**
Modify `latest` parameter to the deprecated version.

### CODE REVIEW

**Comment by kbhawkey** [comment]
Related to #25548.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-25558--k8s-v1-17.netlify.app/docs/reference/generated/kubernetes-api/v1.17/

File: content/en/docs/reference/_index.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ This section of the Kubernetes documentation contains references.
 ## API Reference
 
 * [Kubernetes API Overview](/docs/reference/using-api/api-overview/) - Overview of the API for Kubernetes.
-* [Kubernetes API Reference {{< latest-version >}}](/docs/reference/generated/kubernetes-api/{{< latest-version >}}/)
+* [Kubernetes API Reference {{< param "version" >}}(/docs/reference/generated/kubernetes-api/{{{< param "version" >}}/)
```
Comment (kbhawkey) [inline content/en/docs/reference/_index.md:22]: Whoops. Will fix

**Review by sftim — COMMENTED** [review]
/lgtm
Thanks @kbhawkey

**Comment by kbhawkey** [comment]
I can keep all `latest-version` updates in this PR or let each localization make the fixes.
/cc @kubernetes/sig-docs-zh-reviews @kubernetes/sig-docs-ko-reviews @kubernetes/sig-docs-pl-reviews

Note: I did not replace the `latest-version` in the `contribute` pages.

**Pull request merged**